### PR TITLE
VACMS-18416 Homepage custom analytics removal

### PR DESCRIPTION
**Pull request by randimays**
## Summary
Removed custom analytics from most of the home page's elements.

- Hero banner: "Learn what the PACT Act means for you" link
- Hero banner: "Learn how an account helps you" link
- Hero banner: "Create account" button
- Hero banner fallback: "Learn how an account helps you" link
- Hero banner fallback: "Create an account" button
- Other search tools links
- Top pages links
- News: "VA's mission to better serve all" link
- News: "Read the full article" link
- News: "More VA news" link
- Hub links

Also removed the Teamsites section of the PR template since we don't need to worry about this from a content-build perspective anymore.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18416

## Testing done

Tested the homepage locally using the [Adswerve plugin](https://chromewebstore.google.com/detail/adswerve-datalayer-inspec/kmcbdogdandhihllalknlcjfpdjcleom).

## Screenshots

<details><summary>Hero banner</summary>

### Current hero banner
Screenshot 2024-07-25 at 2 15 42 PM

#### "Learn what the PACT Act means for you" link

Screenshot 2024-07-25 at 12 01 35 PM
Screenshot 2024-07-25 at 12 01 44 PM

#### "Learn how an account helps you" link
Screenshot 2024-07-25 at 12 04 53 PM
Screenshot 2024-07-25 at 12 05 02 PM

### Fallback hero banner (shows if a promo banner is not given)
Screenshot 2024-07-25 at 2 15 38 PM

#### "Learn how an account helps you" link
Screenshot 2024-07-25 at 1 55 29 PM
Screenshot 2024-07-25 at 1 55 39 PM

</details>

<details><summary>"Create account" button</summary>

#### "Create account" button for active promo banner

Screenshot 2024-07-26 at 12 07 59 PM

Screenshot 2024-07-26 at 12 08 09 PM

#### "Create an account" button for fallback promo banner
Screenshot 2024-07-26 at 12 14 18 PM
Screenshot 2024-07-26 at 12 17 13 PM

</details>

<details><summary>Other search tools links</summary>

Screenshot 2024-07-26 at 11 44 36 AM
Screenshot 2024-07-25 at 3 03 01 PM
Screenshot 2024-07-25 at 3 03 14 PM
Screenshot 2024-07-25 at 3 03 24 PM
Screenshot 2024-07-25 at 3 03 35 PM

</details>

<details><summary>Top pages links</summary>
Screenshot 2024-07-26 at 11 46 39 AM

Screenshot 2024-07-25 at 3 04 56 PM
Screenshot 2024-07-25 at 3 05 04 PM
Screenshot 2024-07-25 at 3 05 25 PM
Screenshot 2024-07-25 at 3 05 36 PM
Screenshot 2024-07-25 at 3 05 44 PM

Screenshot 2024-07-25 at 3 05 52 PM
Screenshot 2024-07-25 at 3 05 58 PM
Screenshot 2024-07-25 at 3 06 18 PM
Screenshot 2024-07-25 at 3 08 28 PM
Screenshot 2024-07-25 at 3 08 38 PM

Screenshot 2024-07-25 at 3 08 46 PM
Screenshot 2024-07-25 at 3 08 59 PM

</details>

<details><summary>News section</summary>

Screenshot 2024-07-26 at 11 50 17 AM

#### "VA's mission to better serve all" link
Screenshot 2024-07-25 at 4 28 18 PM
Screenshot 2024-07-25 at 4 28 25 PM

#### "Read the full article" link
Screenshot 2024-07-26 at 11 51 10 AM
Screenshot 2024-07-25 at 4 28 09 PM

#### "More VA News" link
Screenshot 2024-07-25 at 4 24 46 PM
Screenshot 2024-07-25 at 4 24 56 PM

</details>

<details><summary>Hub links</summary>

Screenshot 2024-07-26 at 11 53 50 AM
Screenshot 2024-07-26 at 11 53 50 AM
Screenshot 2024-07-26 at 11 53 58 AM

</details>

### CODE REVIEW

File: src/site/includes/hero.drupal.liquid [hunk 1]
```diff
@@ -87,38 +79,29 @@
             <h1 class="homepage-hero__welcome-headline vads-u-color--white vads-u-padding-y--1 vads-u-border-top--1px vads-u-border-bottom--1px vads-u-font-size--lg vads-u-font-family--serif">
               Welcome to VA.gov
             </h1>
-            <div class="homepage-hero__create-account vads-u-display--none">
+            <div class="homepage-hero__create-account">
```
Comment (chriskim2311) [inline src/site/includes/hero.drupal.liquid:82]: Question: Was this class removal intentional?

Comment (randimays) [reply src/site/includes/hero.drupal.liquid]: Good catch! I removed it so I could see the rest of the fallback banner, thinking it was left there by mistake. But it wasn't! I added it back. Thank you

**Review by chriskim2311 — APPROVED** [review]
LGTM!

**Review by micahchiang — APPROVED** [review]
These web component tags can be self closing, but it's non-blocking.

**Pull request merged**